windows.fnl

```diff
@@ -89,15 +90,39 @@
   (: (hs.window.focusedWindow) :maximize 0)
   (highlight-active-window))
 
+(defn position-window-center
+      [ratio-str window]
```

Comment:
I think the screen should probably be passed in.

Reply:
Maybe make this `position-window-center-on-screen`, though I think the name now is fine.

Reply:
I thought about doing that, but was thinking that really depends on the implementation. The default behavior is to use the active screen, but users may want it to cycle between screens or pick the main screen, or any other logic to select a screen. The ratio-str and window are the only facts that are safe to assume for anticipated use-cases.

Though I suppose it could accept the current screen as a default and not use it but if you are aiming to implement advice then it's kind of a waste to have it query the screens again twice.

Reply:
By making the screen an arg we assuming _less_. We can make `center-window-frame` provide ` (hs.screen.primaryScreen)` by default.

By having more of the knobs in the args it also makes advising easier, since you can write one that just modifies the arguments instead of having to duplicate the logic.

Reply:
You have a good point there. Pushed!